pom.xml

```diff
@@ -1950,6 +1977,12 @@
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
                 <version>${guava.version}</version>
+                <exclusions>
```

Comment:
why do we need this exclusion, because of new guava version?

Reply:
New guava conflicts in some manner with spanner-jdbc:
Here is an exception without this exclusion in Legend Engine - XT - Relational Store - Spanner - Execution - Tests

Dependency convergence error for com.google.errorprone:error_prone_annotations:2.11.0 paths to dependency are:
+-org.finos.legend.engine:legend-engine-xt-relationalStore-spanner-execution-tests:3.21.2-SNAPSHOT
  +-org.finos.legend.engine:legend-engine-xt-relationalStore-executionPlan:3.21.2-SNAPSHOT
    +-com.google.guava:guava:31.1-jre
      +-com.google.errorprone:error_prone_annotations:2.11.0
and
+-org.finos.legend.engine:legend-engine-xt-relationalStore-spanner-execution-tests:3.21.2-SNAPSHOT
  +-org.finos.legend.engine:legend-engine-xt-relationalStore-spanner-execution:3.21.2-SNAPSHOT
    +-com.google.cloud:google-cloud-spanner-jdbc:2.7.3
      +-com.google.errorprone:error_prone_annotations:2.13.1